internal/cli/cloud_manager_clusters_list.go

```diff
@@ -23,23 +23,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type cmClustersListOpts struct {
+type clustersListOpts struct {
```

Comment:
I decided to change the name since this is also used by OM

Reply:
if you want to rename this do it to `cloudManagerClustersListOpts` I'm not that worried about it being shared as they are both the same, at some point `ops-manager` was an alias for `cloud-manager` but we were asked to give it its own space
We may eventually rename all to `opsManager` and files along or like we discussed find a new way to organize these commands (folders/packages?) so feel free of start thinking on those